src/wp-includes/class-wpdb.php

```diff
@@ -3291,6 +3285,109 @@ protected function get_table_charset( $table ) {
 		return $charset;
 	}
 
+	/**
+	 * Retrieves the column metadata for the given column.
+	 *
+	 * @since 6.9.1
```

Comment:
The versions would be 7.0.0.

Reply:
@westonruter Thank you for all of these. I have added all your suggested changes and pushed them.